modules/renter.go

```diff
@@ -314,6 +314,9 @@ type Renter interface {
 	// SetSettings sets the Renter's settings.
 	SetSettings(RenterSettings) error
 
+	// SetDebugFlag sets a debug flag to induce a certain behaviour in the renter
+	SetDebugFlag(flag string)
```

Comment:
This is just something you use in testing, right? Instead of having it be a function of the renter, we probably want to do dependency injection. When you create the renter, you can pass in a dependency like the 'disrupt()' dependency that we have in the contract manager, which will cause the code to behave in strategic ways for certain tests.

This function though is cluttering our specification, and also makes it a lot harder to re-implement the renter for someone who wants to take a stab at re-writing the module in a way that is compatible with the API.

Reply:
Yes, it is. I wanted to create a working testcase with minimal changes + a fix before implementing an injection system like the one in the contract manager. This would also replace the debug flags with a dependency object. Same thing for the wal of course.